react-native/realm.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  appId: '<Your App ID>', // Replace with your App ID
+};
```

Comment:
It might be helpful to throw an error here if the appId has not been set and link to the documentation.

Reply:
Actually I just noticed you can leave the actual app ID you're using here. The backend will automatically replace this file with your app ID. That said, it's still a good idea to make sure it is set somewhere. Also, I'll need to check with Andrew Chen that this format is ok as opposed to a loaded json file.

Reply:
Thanks for feedback guys. I've altered realm.js -> realm.json that looks like this:
```
{
  "appId": "<Your App ID>",
  "url": "http://localhost:8080"
}
```

and added:
```
if (!appId) {
  throw 'Missing Realm App ID. Set your App ID in realm.json';
}
```
where applicable